docs/serverless/08-01-create-function.md

```diff
@@ -62,6 +64,8 @@ Follows these steps:
 
 1. Create a Namespace or select one from the drop-down list in the top navigation panel.
 
+    > **CAUTION:** If you create a new Namespace, do not disable sidecar injection in it as Serverless requires Istio for other resources to communicate with functions correctly. Also, if you apply custom [LimitRanges](https://kyma-project.io/docs/#details-resource-quotas) for a new Namespace, they must be higher than the default values.
```

Comment:
I think that this Caution should be not in tutorial, but in other more often read place like Overview, or Details.

Reply:
yup, I had the same concern but I guess you are right 👍